opacus/accountants/utils.py

```diff
@@ -0,0 +1,46 @@
+from opacus.accountants import get_accountant
+
+
+def get_noise_multiplier(
+    target_epsilon: float,
+    target_delta: float,
+    sample_rate: float,
+    epochs: int,
+    accountant: str = "rdp",
+    **kwargs,
+) -> float:
+    r"""
+    Computes the noise level sigma to reach a total budget of (target_epsilon, target_delta)
+    at the end of epochs, with a given sample_rate
+    Args:
```

Comment:
Extra lines before `Args:` and `Returns:` are probably required to render docs correctly.

Reply:
Let's deal with the docstrings separately